cli-commands/commands.js

```diff
@@ -40,7 +40,7 @@ const addCompileOption = (program) => {
         .command('compile')
         .option('-n --network [network]', 'Network to connect to', "local")
         .option('--path [compile path]', 'Path to contract files', './contracts')
-        .option('--compiler [compiler url]', 'Path to contract files', config.compilerUrl)
+        .option('--compiler [compiler url]', 'Compiling contracts and generating ACI data for contract', config.compilerUrl)
```

Comment:
I thinkg "Url to the desired compiler" will be better, because this is for the current options of the command or I am wrong?